### PR TITLE
A few organizational touches on this project

### PR DESCRIPTION
**Pull request by jetheurer**
I did a few things to this code base that make it a little more organized, and easier for another developer to step in and start code. 

Take a look and feel free to ask questions!  We can do a code review if this is helpful.

I've tried to put in a few comments on what I was doing, but I'm sure some things are not obvious.

Some of the code formatting stuff is from running black: https://black.readthedocs.io/en/stable/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
 # flask_proj05
+
+Install this: https://docs.docker.com/docker-for-mac/install/
```
Comment (kathmbell) [inline README.md:3]: I'm working through all of these comments! Presently working on setting up Docker and figuring out how to run the app using it.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+# this is bunch of pyc files "python compiled" which are generated whenever you run python code.  They should not be included in a repo.
+
+__pycache__/
```
Comment (kathmbell) [inline .gitignore:4]: So this removes / ignores the cached Python files?

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+
+# linting
+Its making sure your code is properly formatted.  using "black" is pretty common now:
+
+```bash
+pip install black
+black flask_proj05/
+```
+
+maybe add some screenshots
```
Comment (kathmbell) [inline README.md]: Add screenshots of the working app?

File: .gitignore (hunk 2 above)
Comment (liz-zither) [reply .gitignore]: Yep, anything in your [.gitignore](https://www.atlassian.com/git/tutorials/saving-changes/gitignore) file is ignored during `git add`, `git commit`, `git push` operations.  We typically ignore "compiled versions of the code as they can be regenerated quite easily by whoever pulls down the repo.  

Also, a common pattern is to have a virtual environment for python (you'll see `venv/bin/activate` around on the web) on your machine.  This virtual environment folder is typically excluded as well.

Finally, for development purposes, you will sometimes want to keep an environmental variable, you can keep track of those environmental variables with a file like `dev.env` and `prod.env`, but put `*.env` in your .gitignore file so you never commit secret variable into the codebase.

Comment (jetheurer) [reply .gitignore:4]: Yep, anything in your [.gitignore](https://www.atlassian.com/git/tutorials/saving-changes/gitignore) file is ignored during `git add`, `git commit`, `git push` operations.  We typically ignore "compiled versions of the code as they can be regenerated quite easily by whoever pulls down the repo.  

Also, a common pattern is to have a virtual environment for python (you'll see `venv/bin/activate` around on the web) on your machine.  This virtual environment folder is typically excluded as well.

Finally, for development purposes, you will sometimes want to keep an environmental variable, you can keep track of those environmental variables with a file like `dev.env` and `prod.env`, but put `*.env` in your .gitignore file so you never commit secret variable into the codebase.

File: README.md (hunk 3 above)
Comment (jetheurer) [reply README.md]: Just an idea; it will make it clearer what this repo does.

**Pull request merged**